trieTools/zeroBalanceSystemAccountChecker/extraTokensCrossChecker.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"errors"
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/tidwall/gjson"
+)
+
+const (
+	maxRequestsRetrial = 10
+	multipleSearchBulk = 10000
+)
+
+type nftBalancesGetter interface {
```

Comment:
maybe you can change the name of the interface? because you also check MetaESDT and Semi Fungible tokens

Reply:
Right, good point, renamed it to "tokenBalancesGetter"